resources/etcd-defrag.timer

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Run etcd-defrag.service periodically
+After=network.target
+
+[Timer]
+OnCalendar=*-*-* 0${hour}:00:00
```

Comment:
```suggestion
OnCalendar=${on_calendar}
```
Could parameterize the whole thing to give us more flexibility to fine tune this?

Reply:
Ah, I see now that it's staggered based on `count.index`.